opal/mca/mpool/grdma/mpool_grdma_module.c

```diff
@@ -213,8 +223,8 @@ bool mca_mpool_grdma_evict (struct mca_mpool_base_module_t *mpool)
 /*
  * register memory
  */
-int mca_mpool_grdma_register(mca_mpool_base_module_t *mpool, void *addr,
-                              size_t size, uint32_t flags,
+int mca_mpool_grdma_register (mca_mpool_base_module_t *mpool, void *addr,
+                              size_t size, uint32_t flags, int8_t access_flags,
```

Comment:
Probably. Just picked the smallest integer type that will hold the flag. I will change to 32.